apps/prairielearn/src/lib/assessment.ts

```diff
@@ -136,7 +137,6 @@ export async function update(
 ): Promise<boolean> {
   debug('update()');
   const updated = await sqldb.runInTransactionAsync(async () => {
-    await sqldb.callAsync('assessment_instances_lock', [assessment_instance_id]);
```

Comment:
This call was unnecessary, since `assessment_instances_update` (next line) also locks the assessment instance.